tests/zn_msgcodec_test.c

```diff
@@ -313,7 +317,7 @@ void assert_eq_payload(_zn_payload_t *left, _zn_payload_t *right)
 void payload_field(void)
 {
     printf("\n>> Payload field\n");
-    _z_wbuf_t wbf = gen_wbuf(128);
+    _z_wbuf_t wbf = gen_wbuf(65535);
```

Comment:
What's the interest of changing `128` to `65535` in the codec test? There were some edge cases triggered by that value? 
Finally, if `65535` is to get the max batch size, shouldn't we use a const for that and make it explicit?

Reply:
`_z_wbuf_t wbf = gen_wbuf(65535);` represents the buffer that will have the serialized message. As such, it should be able to keep the maximum size of the message.

With the new random number generator, serialized message size were surpassing the size of buffer, thus failing the serialization.

Still, applying the suggestion on the const to make it explicit.

Reply:
It sounds good. Thanks for the clarification.